### PR TITLE
Interactive atom definition

### PR DESCRIPTION
**Pull request by wpf500**
Adds an interactive atom definition, see https://github.com/guardian/content-api/pull/1721 for details

### CODE REVIEW

**Comment by wpf500** [comment]
@LATaylor-guardian, could you take a look at this?

File: thrift/src/main/thrift/atoms/interactive.thrift [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace * contentatom.cta
+namespace java com.gu.contentatom.thrift.atom.interactive
+#@namespace scala com.gu.contentatom.thrift.atom.interactive
+
+struct InteractiveAtom {
+  /* the unique ID will be stored in the `atom` data*/
+  2: required string type
```
Comment (LATaylor-guardian) [inline thrift/src/main/thrift/atoms/interactive.thrift]: Why does this start from 2?

File: thrift/src/main/thrift/atoms/interactive.thrift [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  3: required string title,
```
Comment (LATaylor-guardian) [inline thrift/src/main/thrift/atoms/interactive.thrift]: comma not needed.

File: thrift/src/main/thrift/atoms/interactive.thrift (hunk 1 above)
Comment (wpf500) [inline thrift/src/main/thrift/atoms/interactive.thrift]: Not sure, I copied the structure used for the other atom types (e.g. https://github.com/guardian/content-atom/blob/master/thrift/src/main/thrift/atoms/explainer.thrift)

Comment (LATaylor-guardian) [inline thrift/src/main/thrift/atoms/interactive.thrift]: @clloyd Do you know why this is? It exists across other atoms but is not consistent.

Comment (philmcmahon) [inline thrift/src/main/thrift/atoms/interactive.thrift]: I think this is a case of a bad thing being copied twice. I think you're safe to start the numbering at 1.

Comment (clloyd) [inline thrift/src/main/thrift/atoms/interactive.thrift]: So for explainer this stores the type of explainer, there can either be an expandable explainers or just plain explainers.

It's up to the individual atom to decide if there are different types of s specific atom, but it's definitely not required.

Comment (LATaylor-guardian) [inline thrift/src/main/thrift/atoms/interactive.thrift]: @clloyd I'm talking about why the numbering starts from 2 sometimes and other times 1.

Comment (clloyd) [inline thrift/src/main/thrift/atoms/interactive.thrift]: Oh yeah that's rediculous. 1 is correct.

Comment (wpf500) [inline thrift/src/main/thrift/atoms/interactive.thrift]: Okay fixed the numbering

**Comment by LATaylor-guardian** [comment]
These look okay to me 👍  @clloyd and you?

**Comment by clloyd** [comment]
Looks fine to me

**Pull request merged**